client/main.js

```diff
@@ -14,10 +14,16 @@ import AccountSettings from './account-settings';
 import PrintTestLabel from './print-test-label';
 import Packages from './packages';
 import _ from 'lodash';
+import { setNonce, setBaseURL } from 'api/request';
 
-( global.wcConnectData || [] ).forEach( ( wcConnectData ) => {
-	const Route = ( ( rootView ) => {
-		switch ( rootView ) {
+if ( global.wcConnectData ) {
```

Comment:
Note that `wcConnectData` is still used to pass down some properties to the JS bundle, but those properties (`nonce`, `baseUrl`) are specific to WP-Admin.